temporian/core/event_set_ops.py

```diff
@@ -1925,6 +1925,52 @@ def select(
 
         return select(self, feature_names=feature_names)
 
+    def select_index_values(
+        self: EventSetOrNode,
+        keys: Optional[Union[IndexKey, List[IndexKey]]] = None,
+    ) -> EventSetOrNode:
+        """Selects a subset of index values from an
+        [`EventSet`][temporian.EventSet].
```

Comment:
I would add an explanation for advanced users that when using compiled/graph mode, the selected index values are compiled.